test/distributed/org/apache/cassandra/distributed/test/cql3/AccordInteropMultiNodeTableWalkBase.java

```diff
@@ -69,13 +68,6 @@ static void addUncaughtExceptionsFilter(Cluster cluster)
         }
     }
 
-    @Override
-    protected void clusterConfig(IInstanceConfig c)
-    {
-        super.clusterConfig(c);
-        c.set("transaction_timeout", "180s");
```

Comment:
`transaction_timeout` no longer exists, not sure when it was dropped, but confirmed its not present.  We use read/write timeouts, which are already bump to 120s, so its enough to drop this